clients/src/main/java/org/apache/kafka/clients/InFlightRequests.java

```diff
@@ -55,13 +55,20 @@ public void add(ClientRequest request) {
     }
 
     /**
-     * Get the oldest request (the one that that will be completed next) for the given node
+     * Get the oldest request (the one that that will be completed next) for the given node and remove it from the queue
      */
     public ClientRequest completeNext(String node) {
         return requestQueue(node).pollLast();
     }
 
     /**
+     * Get the oldest request (the one that that will be completed next) for the given node
+     */
+    public ClientRequest earliestSent(String node) {
```

Comment:
I agree that it is better to make the names consistent. But in this case, `next` cannot indicate that we are processing the earliest sent request. That is why I used `earliestSent`, which also corresponds to `lastSent`. Do you think we should also change the `completeNext' to 'completeEarlierSent`?